### DISCUSSION TITLE
K0s Install Fail

### DISCUSSION THREAD

**Discussion by liberodark** [tteck/Proxmox#1453] [Help Needed]
### A clear and concise description of the issue.

Hi,

The installation of k0s is broken the script don't inject some fix for run k0s correctly in LXC.
For example `/dev/kmsg`
You need to do a fix to that in LXC settings !
Something like : 

```
lxc.apparmor.profile: unconfined
lxc.cgroup.devices.allow: a
lxc.cap.drop:
lxc.mount.auto: "proc:rw sys:rw"
lxc.mount.entry: /dev/kmsg dev/kmsg none defaults,bind,create=file
```

Best Regards

**Comment by AAH134** [comment]
For me after trying to install,
Everything go through very smooth but not functional k3s because of configurations,

I didnt know what to do to make it work.

If there is anything that must be done to make it functional then U am missing out

**Comment by tteck** [comment]
https://docs.k0sproject.io/v0.9.1/configuration/

`nano /etc/k0s/k0s.yaml`

**Comment by liberodark** [comment]
Before configure you need to fix some issues : 

for example : 

`failed to run Kubelet: failed to create kubelet: open /dev/kmsg: no such file or directory`

@AAH134 thank for your reply but i think you don't have understand the issue. (im not speaking about configuration of k0s)
@tteck thank for configuration file (but i know where it is and is not my the issue)

Best Regards
